sap/5.2/modules/ROOT/pages/sap-connector-studio.adoc

```diff
@@ -0,0 +1,199 @@
++
+See the xref:sap-connector-reference.adoc[SAP Connector Reference] for field values.
+If you chose the *Document source* or *Function source* operation, proceed to the next section to install the SAP libraries.
+. Alternatively, you can configure an input source for this connector such as an *HTTP Listener*, or *Scheduler*, and drag the operation to the Studio canvas. +
+An HTTP Listener enables you to start your app from a browser. A Scheduler starts your app at a specific time.
+. Install the SAP libraries.
+
+== Install SAP Libraries
+
+To use SAP libraries in your Mule project, you must first download the libraries to your
+computer. You can use the three libraries in Anypoint Studio directly, without conversion. You must have an SAP ID to download the libraries.
+
+The file types for each library depend on your operating system:
+
+[%header%autowidth.spread]
+|===
+| OS | File Types
+| Windows a| * sapidoc3.jar
+* sapjco3.jar
+* sapjco3.dll
+
+| Mac OS a| * sapidoc3.jar
+* sapjco3.jar
+* libsapjco3.jnilib
+
+| Linux a| * sapidoc3.jar
+* sapjco3.jar
+* libsapjco3.so
+|===
+
+[NOTE]
+SAP Connector supports only version 3.0.x SAP JCo libraries.
+
+After you install the libraries, you can customize the global element properties and test the connection.
+
+== Configure the Libraries in Anypoint Studio
+
+After installing the SAP Connector in Studio and adding the connector to your Mule project, configure the libraries.
+
+. Double-click the connector in the flow.
+. Click the plus sign (*+*) on the right of the *Connector Configuration* field. +
+This screen appears:
++
+image::sap-connector-configuration-01.png[]
+
+=== Install the IDOC Library
+
+. Click *Add Dependency*.
+. Click *Install*.
++
+image::sap-connector-configuration-02.png[]
++
+. In the *Install* menu, click *Browse* to locate the library file.
+. Specify a unique group ID, artifact ID, and version.
++
+image::sap-connector-configuration-02-in.png[]
++
+. Click *Finish* or, if you want to cancel, click *Remove Dependency*.
++
+image::sap-connector-configuration-02-finish.png[]
+
+=== Install the JCO Library
+
+. Click *Add Dependency*.
+. Click *Install*.
++
+image::sap-connector-configuration-02.png[]
++
+. In the *Install* menu, click *Browse* to locate the library file.
+. Specify a unique group ID, artifact ID, and version.
++
+image::sap-connector-configuration-03-in.png[]
++
+. Click *Finish* or, if you want to cancel, click *Remove Dependency*.
++
+image::sap-connector-configuration-03-finish.png[]
+
+=== Install the JCO Native Library
+
+. Click *Add Dependency*.
+. Click *Install*.
++
+image::sap-connector-configuration-02.png[]
++
+. In the *Install* menu, click *Browse* to locate the library file.
++
+On MacOS, starting with JCO version 3.1.2, the native library extension was changed from `jnilib` to `dylib`. Anypoint Studio versions 7.0 through 7.6 have a limitation on selecting the `dylib` file type.
```

Comment:
I think we need to maybe reword this, something like: Starting with Anypoint Studio 7.0, there is a limitation..."
Because Studio is now on version 7.7. Maybe this limitation doesn't apply to 7.7? Needs to be verified with dev.